src/app/funding/funding.component.ts

```diff
@@ -63,13 +68,44 @@ export class FundingComponent {
     altImageText: 'CFI Logo',
     content: `The authors wish to acknowledge the funding support of the Canada Foundation for Innovation Cyberinfrastructure Initiative, the Ontario Research Fund, BC Knowledge Development Fund and the Ministère de l'Économie, de la Science et l'Innovation for 'The Cancer Genome Collaboratory' project.`,
   };
+  AmazonFunder: Funder = {
+    title: 'Amazon',
+    website: 'https://aws.amazon.com/genomics-cli/',
+    imageSource: '../assets/images/sponsors/aws.png',
+    altImageText: 'AWS Logo',
+    content: `Amazon has partnered with Dockstore to allow Amazon Genomics CLI users quick and easy access to the workflows registered with Dockstore.`,
+  };
+  NHGRIFunder: Funder = {
+    title: 'NHGRI',
+    website: 'https://www.genome.gov/',
+    imageSource: '../assets/images/sponsors/NHGRI.png',
+    altImageText: 'NHGRI Logo',
+    content: `NHGRI is part of the National Institute of Health (NIH). As a leading authority in the field of genomics, NHGRI's mission is to accelerate scientific and medical breakthroughs that improve human health. We do this by driving cutting-edge research, developing new technologies, and studying the impact of genomics on society.`,
+  };
+
   funders: Funder[] = [
     this.GenomeCanadaFunder,
     this.ProvinceOfOntarioFunder,
-    this.NIHFunder,
     this.BISTIFunder,
+    this.NIHFunder,
+    this.NHGRIFunder,
     this.BioDataCatalyst,
     this.CFIFunder,
+    this.AmazonFunder,
+  ];
+
+  currentFunders: Funder[] = [this.AmazonFunder, this.GenomeCanadaFunder, this.NHGRIFunder, this.NIHFunder, this.ProvinceOfOntarioFunder];
```

Comment:
They are no longer a current funder, correct @bvizzier ?

Reply:
AWS. Sorry, It was in my mind, but I forgot to write it down.

Reply:
Sorry, what is the resolution to this? Do we keep them in currentFunders or not?